.github/workflows/release.yml

```diff
@@ -12,20 +12,19 @@ permissions:
 jobs:
   release-please:
     permissions:
-      contents: write # for googleapis/release-please-action to create release commit
+      id-token: write # for googleapis/release-please-action to create release tag
+      contents: write # for googleapis/release-please-action to create release commit, SBOM release
       pull-requests: write # for googleapis/release-please-action to create release PR
       packages: read # for internal nuget reading
-      
+      attestations: write # for actions/attest-sbom to create attestation
+
     runs-on: ubuntu-latest
 
     steps:
-      - uses: googleapis/release-please-action@db8f2c60ee802b3748b512940dde88eabd7b7e01 # v3
+      - uses: googleapis/release-please-action@a02a34c4d625f9be7cb89156071d8567266a2445 #v4
```

Comment:
I'm in favor of switching to v4 but we'll need to be more defensive in the `if` conditions to ensure everything continues to work. FYI @toddbaert

Reply:
@beeme1mr Do you have any suggestions on where I should add the `if` conditions in this action?

Reply:
Something like this should work.

https://github.com/open-feature/java-sdk-contrib/blob/main/.github/workflows/release-please.yml#L29